tests/misc/CachingTest.py

```diff
@@ -146,13 +146,11 @@ def test_caching_multi_results(self):
                 retval, output = execute_coala(
                    coala.main,
                    'coala',
-                   '-c', filename + '.coafile',
-                   '-f', filename + 'test.py')
+                   '-c', filename + '.coafile')
```

Comment:
As discussed on gitter, this would also apply to `-m`, which means [this PR](https://github.com/processing/processing/pull/4781/files) would fail after 0.10 docker is released